core/metacling/src/TClingCallbacks.cxx

```diff
@@ -84,16 +84,22 @@ extern "C" {
 }
 
 class AutoloadLibraryMU : public llvm::orc::MaterializationUnit {
+   const TClingCallbacks &CB;
```

Comment:
To reduce the diff, it would have been easier to move the new member next to the existing ones. IIRC I'm following upstream style here by having the members last in each `class`...